test/gerkin/features/steps/data_update_entries.js

```diff
@@ -0,0 +1,150 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { header, localhost } = require('./helpers/helpers');
+
+let userVariables = {};
+let specDataUpdateEntries;
+
+const baseUrl = `${localhost}data/registry1/version1/update-entries`;
+
+Before(() => {
+  specDataUpdateEntries = pactum.spec();
+});
+
+// Scenario: The user successfully updates two existing records in the Digital Registries database
+Given(
+  'The user wants to update multiple records with first name "Alfie" in the Digital Registries database',
+  () => {
+    userVariables = {
+      ID: 'EE378627348834',
+      FirstName: 'Alfie',
+      LastName: 'Grande',
+      BirthCertificateID: 'RR-1234567889',
+    };
+
+    return userVariables;
+  }
+);
+
+Given('The records with first name "Alfie" exist in the database', () => {
+  return 'The records with first name "Alfie" exist in the database';
+});
+
+When(
+  'The user triggers an action to update records with first name "Alfie" in the database',
+  () => {
+    specDataUpdateEntries
+      .put(`${baseUrl}`)
+      .withHeaders(`${header.key}`, `${header.value}`)
+      .withBody({
+        query: {
+          content: userVariables,
+        },
+        write: {
+          content: userVariables,
+        },
+      });
+  }
+);
+
+Then(
+  'Operation to update records with first name "Alfie" finishes successfully',
+  async () => {
+    await specDataUpdateEntries.toss();
+    specDataUpdateEntries.response().should.have.status(200);
+  }
+);
+
+// Scenario: The user is not able to update two records which do not exist in the Digital Registries database
+Given(
+  'The user wants to update multiple records with first name "Jerry" in the Digital Registries database',
+  () => {
+    userVariables = {
+      ID: 'EE378627348834',
+      FirstName: 'Jerry',
+      LastName: 'Blake',
+      BirthCertificateID: 'RR-1234567889',
+    };
+
+    return userVariables;
+  }
+);
+
+Given(
+  'The records with first name "Jerry" do not exist in the database',
+  () => {
+    return 'The records with first name "Jerry" do not exist in the database';
+  }
+);
+
+When(
+  'The user triggers an action to update records with first name "Jerry" in the database',
+  () => {
+    specDataUpdateEntries
+      .put(`${baseUrl}`)
```

Comment:
Just a quick note about using PUT HTTP method - when used in update context it is meant to overwrite all attributes of record in the database (not overwrite part of the fields as in PATCH). Are we sure that's an desired behavior and  should add test case to check if it's used correctly?